include/volume/volume_cooling_balls.hpp

```diff
@@ -19,7 +19,9 @@
 #include "convex_bodies/zpolytope.h"
 #include "convex_bodies/ballintersectconvex.h"
 #include "convex_bodies/vpolyintersectvpoly.h"
-#include "volume/rounding.hpp"
+#include "preprocess/mee_rounding.hpp"
```

Comment:
do the volume algorithms actually need the rounding algorithms?

Reply:
no, only in `volume_sequence_of_hpoly.hpp`

Reply:
ok so those includes are not needed

Reply:
Yes of course, removed.